src/main/java/uk/gov/justice/hmpps/prison/service/BookingService.java

```diff
@@ -1119,7 +1120,12 @@ private Sort mapBookingSortOrderProperties(Sort sort) {
     }
 
     @VerifyBookingAccess(overrideRoles = {"SYSTEM_USER", "GLOBAL_SEARCH", "VIEW_PRISONER_DATA"})
-    public List<Prison> getBookingVisitsPrisons(final Long bookingId) {
-        return visitInformationRepository.findByBookingIdGroupByPrisonId(bookingId);
+    public List<PrisonDetails> getBookingVisitsPrisons(final Long bookingId) {
+        return visitInformationRepository.findByBookingIdGroupByPrisonId(bookingId)
+            .stream().map(prison -> PrisonDetails.builder()
+                .prisonId(prison.getPrisonId())
+                .prison(LocationProcessor.formatLocation(prison.getPrisonDescription()))
+                .build())
+            .collect(Collectors.toList());
```

Comment:
meh.. don't you get a warning around just using `Stream.toList()` instead?

Reply:
I didn't but will take a look